com.jianguoyun.Nutstore.yaml

```diff
@@ -0,0 +1,87 @@
+app-id: com.jianguoyun.Nutstore
+runtime: org.gnome.Platform
+runtime-version: "46"
+sdk: org.gnome.Sdk
+command: nutstore
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # So that we can sync files anywhere we want.
+  - --filesystem=home
+separate-locales: false
+
+modules:
+  # Dependency of webkitgtk.
+  - name: unifdef
+    buildsystem: simple
```

Comment:
This should be cleaned up, it's a build time dependency only.

```yml
cleanup:
  - '*'
```

Reply:
Done.